docs/content/develop/templates.md

```diff
@@ -0,0 +1,280 @@
++++
+title = "Templates & slots"
+description = ""
+weight = 3
+draft = false
+bref = ""
+toc = true
+menu = "start"
+tags = [ "start" ]
++++
+
+
+[TOC]
+
+# Light DOM vs. Shadow DOM
+
+The concept of light DOM is already familiar to any web developer. Basic HTML elements that you place into a `.html` file are light DOM. Tags like `<h1>`, `<div>` and `<aside>` are all simple light DOM. 
+
+However, when you add a tag to the light DOM such as `select`, the browser has additional styles that are invoked from within the Shadow DOM. 
+
+When building web components, you may choose whether to keep content in the light DOM (i.e. pfe-tab), or to copy it from the light DOM into the shadow DOM (i.e. pfe-accordion-header), or to place it in the shadow DOM only (i.e. close button on pfe-modal).
+
+Consider the pros and cons for each approach, and remember that web components should still contain semantic markup and fail gracefully, so that users or search crawlers can still understand the content on the page.
+
+
+## Light DOM
+
+
+```
+  <select>
+    <option value="books">Books</option>
+    <option value="html">HTML</option>
+    <option value="css">CSS</option>
+    <option value="php">PHP</option>
+    <option value="js">JavaScript</option>
+  </select>
+```   
+
+ <select>
+    <option value="books">Books</option>
+    <option value="html">HTML</option>
+    <option value="css">CSS</option>
+    <option value="php">PHP</option>
+    <option value="js">JavaScript</option>
+ </select>
+  
+ 
+![HTML select tag](/HTML__select__Tag.jpg)
+
+One part of web components is the ability to utilize the Shadow DOM to store additional markup. There are pros and cons to placing things in the Light DOM: 
+
+### **Pros**
+
+1. Search engines can see content, resulting in better SEO
+2. Users with JavaScript disabled can see content
+3. Content can be themed with normal page stylesheets
+    1. Sometimes this is desirable, in the case of paragraphs and links inside a pfe-tab panel for instance.
+
+
+### **Cons**
+
+1. Web component styles can conflict with styles already on the page, and existing styles will probably be more specific than `::slotted` foo
+
+    ```
+    // regular-styles.css
+    pfe-cta a {
+	    color: foo;
+    }
+    ```
+    
+    is more specific than  
+     
+	```
+	// web-component.css
+	::slotted(a) {
+      color: foo !important;
+   }
+   ```
+2. Internet Explorer & Edge
+    - There are no slots in IE11 or Edge, so the polyfill **will not**  style slotted elements.
```

Comment:
"Elements that can be inserted into slots are known as Slotable; when an element has been inserted in a slot, it is said to be slotted." https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots

Reply:
So you are not really styling slots in IE11. You can still target the elements that are slotted (in other browsers) in various other ways in IE11, by using other selectors. but `::slotted {}` won't help you.

Reply:
IE11 gets complicated - if you add classes to elements in the component's template, you actually can style that class and it does work if the component upgrades correctly.

Reply:
It's really hard to detect because it doesn't show up well in the inspector but if you select a shadow-element that has a custom class on it, it will show up in the header region above the body and it does get styles.

Reply:
<img width="745" alt="Screen Shot 2019-09-17 at 3 05 54 PM" src="https://user-images.githubusercontent.com/1840295/65071397-b21d6400-d95c-11e9-9719-0f2135e0cbd5.png">

Reply:
<img width="1366" alt="Screen Shot 2019-09-17 at 3 08 15 PM" src="https://user-images.githubusercontent.com/1840295/65071545-ff99d100-d95c-11e9-99ac-735ca0e4aa07.png">

Another good screenshot with a more complex band layout ^

Reply:
So maybe we remove this from the doc for now and add a TODO to discover this further? cc @starryeyez024 @kylebuch8

Reply:
Note: will add language that Shady CSS polyfills ::slotted(). Its not _really_ slotted but its a workaround so that you can use that selector